### PR TITLE
[Core] FilterForm cleanup

### PR DESCRIPTION
**Pull request by alisterdev**
- [x] Make hardcoded properties dynamic
- [x] Remove unused filter class property

### CODE REVIEW

File: jsx/FilterForm.js [hunk 1]
```diff
@@ -162,7 +162,11 @@ class FilterForm extends React.Component {
     }
 
     return (
-      <Panel id="selection-filter" title="Selection Filter">
+      <Panel
+        id={this.props.id}
+        height={this.props.height}
+        title={this.props.title}
```
Comment (jacobpenny) [inline jsx/FilterForm.js]: We should add these props (id, height, and title) to `FilterForm.propTypes`

Comment (alisterdev) [inline jsx/FilterForm.js]: @jacobpenny they are part of Panel.propTypes so, all the warning will be triggered anyways

Comment (jacobpenny) [inline jsx/FilterForm.js]: But won't the warning point to the inner component? It's useful to have the type checking at both levels.
It also acts as documentation. As a client of `FilterForm` I can see what I can pass as a prop without digging into the implementation.

Comment (alisterdev) [inline jsx/FilterForm.js]: @jacobpenny Fair enough, fixed!

**Pull request merged**